### PR TITLE
Upgrade/800/ethernet networks and uplink sets

### PR DESCRIPTION
**Pull request by sijeesh**
### Description
Upgraded Ethernet and Uplinkset resources to support OneView API version 800

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass for Python 2.7+ & 3.4+(`$ tox`).
- [ ] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [x] New endpoints supported are updated in the endpoints-support.md file.
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: hpOneView/resources/networking/uplink_sets.py [hunk 1]
```diff
@@ -47,120 +48,16 @@ class UplinkSets(object):
...
-            timeout:
-                Timeout in seconds. Wait for task completion by default. The timeout does not abort the operation in
-                OneView, just stops waiting for its completion.
-
-        Returns:
-            dict: Created resource.
-        """
-        return self._client.create(resource, timeout=timeout, default_values=self.DEFAULT_VALUES)
-
-    def update(self, resource, timeout=-1):
```
Comment (soodpr) [inline hpOneView/resources/networking/uplink_sets.py]: here we need to add query param: force

Comment (sijeesh) [reply hpOneView/resources/networking/uplink_sets.py]: Update in base class already have a force param.
https://github.com/HewlettPackard/python-hpOneView/blob/master/hpOneView/resources/resource.py

Comment (soodpr) [reply hpOneView/resources/networking/uplink_sets.py]: yes, but default value is false but in API800 spec, its true

File: examples/ethernet_networks.py [hunk 2]
```diff
@@ -31,7 +31,8 @@
     "credentials": {
         "userName": "<username>",
         "password": "<password>"
-    }
+    },
+    "api_version": 800
```
Comment (madhav-bharadwaj) [inline examples/ethernet_networks.py]: Can we give a placeholder for this also?

Comment (sijeesh) [reply examples/ethernet_networks.py]: I think it's better to keep the latest API version as we have an updated resource module and example file to support the latest version of the API.

File: hpOneView/resources/networking/uplink_sets.py (hunk 1 above)
Comment (soodpr) [reply hpOneView/resources/networking/uplink_sets.py]: ok. I am approving the PR as of now and lets send out an email to OneView team for clarification.

Comment (sijeesh) [reply hpOneView/resources/networking/uplink_sets.py]: Sure, This has to be reported, let's release it now with the default value as False.

**Pull request merged**